### PR TITLE
puppet-lint: fix strict_indent

### PR DESCRIPTION
**Pull request by bastelfreak**
this check comes from a new plugin:
https://github.com/relud/puppet-lint-strict_indent-check

if this works fine I would like to add it as new dependency to
voxpupuli-test.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ end
 
 group :test do
   gem 'voxpupuli-test', '>= 1.0.0',  :require => false
+  gem 'puppet-lint-strict_indent-check', :require => false
```
Comment (bastelfreak) [inline Gemfile:15]: I didn't adjust the indent here because it will be purged by the next modulesync anyways.

File: manifests/install.pp [hunk 2]
```diff
@@ -13,7 +13,7 @@
   }
 
   if $collectd::utils and  ( $facts['os']['family'] == 'Debian' or
-    ( $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'8') >= 0 )) {
+  ( $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'8') >= 0 )) {
```
Comment (ekohl) [inline manifests/install.pp:16]: This one looks odd to me

File: manifests/plugin/perl/plugin.pp [hunk 3]
```diff
@@ -71,7 +71,7 @@
       }
     }
     default: {
-      fail("Unsupported provider: ${provider}. Use 'package', 'cpan',
+    fail("Unsupported provider: ${provider}. Use 'package', 'cpan',
```
Comment (ekohl) [inline manifests/plugin/perl/plugin.pp:74]: This also looks odd

**Comment by bastelfreak** [comment]
@ekohl fixed all

File: manifests/plugin/disk.pp [hunk 4]
```diff
@@ -17,9 +17,9 @@
       $_manage_package = $manage_package
     } else {
       if versioncmp($collectd::collectd_version_real, '5.5') >= 0
-        and versioncmp($facts['os']['release']['major'],'8') >= 0 {
+      and versioncmp($facts['os']['release']['major'],'8') >= 0 {
```
Comment (ekohl) [inline manifests/plugin/disk.pp:20]: This also looks incorrect. I think that's a bug in the actual lint plugin on multi line if conditions. Perhaps it needs braces?

Comment (bastelfreak) [reply manifests/plugin/disk.pp:20]: yes it need braces. I will add them and raise an issue upstream

Comment (bastelfreak) [reply manifests/plugin/disk.pp:20]: not sure if this an actual bug in the Puppet DSL. the code still compiles correctly?

Comment (ekohl) [reply manifests/plugin/disk.pp:20]: AFAIK Puppet DSL doesn't care about whitespace at all.

**Comment by ekohl** [comment]
This was included in modulesync 3.0.0.

**Pull request closed without merging**